docs/cn/Command-Line-Interface.md

```diff
@@ -8,35 +8,36 @@ priority: 0
 * 内容列表
 {:toc}
 
-Alluxio命令行接口为用户提供了基本的文件系统操作，可以使用以下语法调用命令行功能：
-
-{% include Command-Line-Interface/alluxio-fs.md %}
-
-fs命令中的所有“路径”都应该以以下开头：
-
-{% include Command-Line-Interface/alluxio-path.md %}
-
-或者如果没有提供路径前缀，默认的主机名和端口（见env文件）将会被使用。
-
-    /<path>
-
-## 通配符输入
-
-大多数需要路径参数的命令可以使用通配符以便简化使用，例如：
-
-{% include Command-Line-Interface/rm.md %}
-
-该示例命令会将`data`文件夹下以`2014`为文件名前缀的所有文件删除。
-
-注意有些shell会尝试自动补全输入路径，从而引起奇怪的错误（注意：以下例子中的数字可能不是21，这取决于你的本地文件系统中匹配文件的个数）：
-
-{% include Command-Line-Interface/rm-error.md %}
-
-作为一种绕开这个问题的方式，你可以禁用自动补全功能（跟具体shell有关，例如`set -f`），或者使用转义通配符，例如：
-
-{% include Command-Line-Interface/escape.md %}
-
-注意是两个转义符号，这是因为该shell脚本最终会调用一个java程序运行，该java程序将获取到转义输入参数（cat /\\*）。
+Alluxio命令行接口为用户提供了基本的文件系统操作，可以使用以下命令来得到所有子命令：
+
+```bash
+$ ./bin/alluxio fs
+Usage: alluxio fs [generic options]
```

Comment:
These changes should go into `Command-Line-Interface/alluxio-fs.md`, so it can be used in all languages.